RosBE-Windows/Root/ChangeLog.txt

```diff
@@ -1,3 +1,13 @@
+*** Jun xxth, 2021 - RosBE 2.3.0 Released
```

Comment:
2021?

Reply:
Now you know how long this stuff way rotting already. Will be fixed later.